iocore/net/P_SSLNetVConnection.h

```diff
@@ -259,6 +260,12 @@ class SSLNetVConnection : public UnixNetVConnection
   /// Set by asynchronous hooks to request a specific operation.
   SslVConnOp hookOpRequested;
 
+  void
+  clearnpnSet()
```

Comment:
The naming should be consistent with `registerNextProtocolSet` since they are inverses of each other. In fact, it might be reasonable to have only one and pass `nullptr` to clear it (e.g., just `setNextProtocolSet`).